src/main/java/org/jvnet/hudson/plugins/platformlabeler/PlatformLabelerGlobalConfiguration.java

```diff
@@ -26,6 +26,7 @@ public PlatformLabelerGlobalConfiguration() {
   @Override
   public boolean configure(StaplerRequest req, JSONObject json) throws FormException {
     boolean result = super.configure(req, json);
+    java.util.logging.Logger.getLogger("foo").warning(req.getHeader("foo"));
```

Comment:
Generate a fake findsecbugs finding. Otherwise, it's difficult to make sure it's working and to demonstrate suppression.